### ISSUE TITLE
Issue when Browser debugging in VSCode

### ISSUE THREAD

**Issue by grbd** [nuxt/content#1639]
### Environment

------------------------------
- Operating System: `Windows_NT`
- Node Version:     `v18.1.0`
- Nuxt Version:     `3.0.0-rc.12`
- Nitro Version:    `0.6.0`
- Package Manager:  `yarn@3.2.3`
- Builder:          `vite`
- User Config:      `modules`, `content`, `hooks`, `tailwindcss`, `components`, `colorMode`
- Runtime Modules:  `@nuxt/content@2.2.0`, `@nuxtjs/tailwindcss@^6.1.1`, `@nuxtjs/color-mode@^3.1.8`
- Build Modules:    `-`
------------------------------

### Reproduction

Debugging the browser window for a nuxt content application under VSCode with Windows

### Describe the bug

Hi,
Not sure if this is fixable but I've noticed when debugging a nuxt 3 / content 2 app with VSCode.
The browser has trouble connecting back to the server I think for tracking changes to the markdown.
```
web-socket.mjs?v=063752cb:5 [Content] WS Error: Event {isTrusted: true, type: 'error', target: WebSocket, currentTarget: WebSocket, eventPhase: 2, …}
warn @ web-socket.mjs?v=063752cb:5
onError @ web-socket.mjs?v=063752cb:30
error (async)
connect @ web-socket.mjs?v=063752cb:59
setTimeout (async)
connect @ web-socket.mjs?v=063752cb:44
onClose @ web-socket.mjs?v=063752cb:38
web-socket.mjs?v=063752cb:4 [Content] WS reconnecting..
web-socket.mjs?v=063752cb:4 [Content] WS connect to ws://[::]:24678/ws
web-socket.mjs?v=063752cb:56 WebSocket connection to 'ws://[::]:24678/ws' failed: Error in connection establishment: net::ERR_ADDRESS_INVALID
```

Example of .vscode/launch.json
```json
{
    "version": "0.2.0",
    "configurations": [
      {
        "name": "Launch Browser Chrome",
        "request": "launch",
        "type": "chrome",
        "url": "http://localhost:3000",
        "webRoot": "${workspaceFolder}",
      },
      {
        "name": "Launch Browser Firefox",
        "request": "launch",
        "type": "firefox",
        "url": "http://localhost:3000",
        "webRoot": "${workspaceFolder}"
      },
      {
        "name": "nuxt-server",
        "request": "launch",
        "type": "node",
        "cwd": "${workspaceFolder}",
        "runtimeExecutable": "yarn",
        "runtimeArgs": ["dev"],
        "console": "integratedTerminal"
      },
    ],
    "compounds": [
      {
        "name": "Server/Client",
        "configurations": ["nuxt-server", "Launch Browser Chrome"],
        "stopAll": true
      }
    ]
}
```

It's not a big issue as I can just switch the content watch off while debugging vue controls
Then back on to track changes to markdown.
```
  content: {
    watch: false,
  },
```

But it might be worth a mention in the docs

**Comment by farnabaz** [comment]
This issue should be fixed in #1624 and published in edge-channel. Also next version with fix will release soon

**Comment by farnabaz** [comment]
Released in the latest version
